src/metabase/pulse/render/body.clj

```diff
@@ -555,7 +555,7 @@
                                     (or (<= min-a min-b max-a)
                                         (<= min-a max-b max-a)))]
     (if
-      overlapping-and-valid?
+     overlapping-and-valid?
```

Comment:
My Calva extension also formats the `if` macro like this, but I checked other places that use this style—if then immediately follows by a line break—the next line should be indented with 2 spaces i.e. reverting this change would make the style more consistent.